src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
Note that we _do_ have the information of the alignment already (it's on the type's `layout`), why do we need to rely on parsing the attribute ourselves?